src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="bigForm"
+      action="https://localhost:3000/api"
+      method="get"
+      data-qa="big"
+    >
+      <img
+        src="images/Search.svg"
+        alt="search"
+        class="icon"
+      >
+
+      <input
+        class="input"
+        type="text"
+        name="searchBarBig"
+        id="searchBarBig"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        required
+      >
+    </form>
+
+    <form
+      class="smallFrom"
+      action="https://localhost:3000/api"
+      method="get"
+      data-qa="small"
+    >
+      <img
+        src="images/Search.svg"
+        alt="search-bar-small"
+        class="icon icon--small"
+      >
+
+      <input
+        class="input-small"
```

Comment:
After this changes you can remove your repeatable class usage
Fix all cases
![image](https://user-images.githubusercontent.com/70009205/213700007-2002de00-1a66-4301-bfd8-52f0b3413ccd.png)

```suggestion
        class="input input-small"
```